lib/ansible/module_utils/network/ios/utils/utils.py

```diff
@@ -0,0 +1,155 @@
+#
+# -*- coding: utf-8 -*-
+# Copyright 2019 Red Hat
+# GNU General Public License v3.0+
+# (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+# utils
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+from ansible.module_utils.six import iteritems
+
+
+def remove_command_from_interface(interface, cmd, commands):
```

Comment:
The function doesn't seem specific to the interface. Probably can be renamed to a more generic name something like remove_command_from_list(), add_command_to_list()